static/app/utils/fields/index.ts

```diff
@@ -251,6 +254,21 @@ export const AGGREGATION_FIELDS: Record<AggregationKey, FieldDefinition> = {
     kind: FieldKind.FUNCTION,
     valueType: FieldValueType.NUMBER,
   },
+  [AggregationKey.SPS]: {
+    desc: t('Spans per second'),
+    kind: FieldKind.FUNCTION,
+    valueType: FieldValueType.NUMBER,
+  },
+  [AggregationKey.TIME_SPENT_PERCENTAGE]: {
+    desc: t('Span time spent percentage'),
+    kind: FieldKind.FUNCTION,
+    valueType: FieldValueType.NUMBER,
+  },
+  [AggregationKey.HTTP_ERROR_COUNT]: {
+    desc: t('Count of 5XX http errors'),
+    kind: FieldKind.FUNCTION,
+    valueType: FieldValueType.NUMBER,
+  },
```

Comment:
This is a really good point! I think i'm just gonna code it up so it doesn't appear in discover at all.

Reply:
So I updated this, couldn't find an obvious way to just hide it in the UI.
Instead i'm using a new variable `STARFISH_AGGREGATION_FIELDS`, this closely resembles the AGGREGATIONS in discover fields so that we can easily move it over if we want it visible everywhere. lmk if you have any thoughts!